ci/README.md

```diff
@@ -38,9 +38,8 @@ running tests that reflects the latest built Windows 2019 Envoy image.
 
 # Build image base and compiler versions
 
-Currently there are three build images for Linux and one for Windows:
+Currently there are two build images for Linux and one for Windows:
```

Comment:
there is one for linux and none for windows

the only image we currently maintain is envoy-build-ubuntu